client/src/Backoffice/components/Main/components/modals/AddCustomer/AddCustomerModal.tsx

```diff
@@ -29,6 +30,7 @@ interface AddCustomerModalProps {
 }
 function AddCustomerModal({ open, handleModal }: AddCustomerModalProps) {
   const [activeStepIndex, setActiveStepIndex] = useState(0);
+  const [existingEmails, setExistingEmails] = useState<Set<string>>(new Set());
```

Comment:
Why it's need to be a set and not simply a String value?

Reply:
Because in case you try two (or more) attempts with two existing emails, it'll show you the error for both emails. If it was a string, say you tried a first existing email, and then tried another existing one the validator will 'forget' about the first one. It is nicer this way.